commit 227cba74b84c55c3d8677f0c10e3f8e35c84e04e
Author: Fernando Vilas <fvilas@iname.com>
Date:   Fri Mar 2 22:25:43 2012 -0600

    BUG: 295235
    
    Fix crash on opening SQLite database.
    (cherry picked from commit 6af4b33df2476f568db852d96b99eb45eaa2e21f)

diff --git a/kmymoney/mymoney/storage/mymoneystoragesql.cpp b/kmymoney/mymoney/storage/mymoneystoragesql.cpp
index ab5ed4c..4a170b1 100644
--- a/kmymoney/mymoney/storage/mymoneystoragesql.cpp
+++ b/kmymoney/mymoney/storage/mymoneystoragesql.cpp
@@ -3348,8 +3348,6 @@ const QMap<QString, MyMoneySchedule> MyMoneyStorageSql::fetchSchedules(const QSt
   if (forUpdate)
     queryString += m_driver->forUpdateString();
 
-  queryString += ';';
-
   q.prepare(queryString);
 
   if (! idList.empty()) {
